js/wp-seo-replacevar-plugin-302.js

```diff
@@ -42,11 +53,68 @@
 			data = this.parentReplace( data );
 			data = this.doubleSepReplace( data );
 			data = this.excerptReplace( data );
+			data = this.categoryReplace( data );
+			data = this.customTaxonomyReplace( data );
+			data = this.tagReplace( data );
+
+			// Taxonomy replacements
+			if ( jQuery( 'input[name=tag_ID]').val() !== undefined ) {
+				data = this.taxonomyDescriptionReplace( data );
+			}
 		}
 		return data;
 	};
 
 	/**
+	 * Binding events for each taxonomy metabox element.
+	 *
+	 * @param {int} index
+	 * @param {Object} taxonomyMetaBox
+	 */
+	YoastReplaceVarPlugin.prototype.setTaxonomyEvents = function( index, taxonomyMetaBox ) {
+		// Set the events.
+		taxonomyMetaBox = jQuery( taxonomyMetaBox );
+
+		taxonomyMetaBox.on( 'wpListAddEnd', '.categorychecklist', function() {
+			this.fetchTaxonomies( taxonomyMetaBox );
```

Comment:
This can be simplified to:
```js
taxonomyMetaBox.on( 'wpListAddEnd', '.categorychecklist', this.fetchTaxonomies.bind( this, taxonomyMetaBox ) );
```